src/tests/tracing/eventpipe/buffersize/buffersize.csproj

```diff
@@ -7,6 +7,7 @@
     <CLRTestPriority>0</CLRTestPriority>
     <UnloadabilityIncompatible>true</UnloadabilityIncompatible>
     <JitOptimizationSensitive>true</JitOptimizationSensitive>
+    <GCStressIncompatible>true</GCStressIncompatible>
```

Comment:
Do we understand why this test is GCStress incompatible?

Reply:
See the failure [here](https://github.com/dotnet/runtime/issues/39171).  My understanding is that it is a timing issue induced by gcstress.  The reader is not keeping up with events and is dropping some 3000 events.  If you look at the times in the log, the entirety of the events being sent appear to happen in the same `0.1` seconds.  There are only 1000 events sent, so I'm inclined to believe that the reader dropped all the test events considering it only saw 544 rundown events.